src/firebase.ts

```diff
@@ -1,7 +1,14 @@
 import * as firebase from 'firebase-admin';
 
+// Providing a path to a service account key JSON file
 firebase.initializeApp({
-    credential: firebase.credential.cert(JSON.parse(process.env.FIREBASE_KEY!))
+    credential: firebase.credential.cert({
+        'privateKey': process.env.FIREBASE_PRIVATE_KEY,
+        'projectId': process.env.FIREBASE_PROJECT_ID,
+        'clientEmail': process.env.FIREBASE_CLIENT_EMAIL,
+    }),
```

Comment:
Does this need changing from using the json string which contains this information?

Reply:
@eddiejaoude yeah, I was wondering what the community felt about this because the "firebase certificate" has one key (can't remember which one) which breaks when using `admin.authentication.cert` when you export a service worker.

I thought it'd be even tidier for when going to CI for anybody because they can use environment variables instead of Base64ing a string or JSON Stringify etc etc

Reply:
Ok, I will need to update things on my side before I merge this. I can't wait to have this, it will be a step closer to removing Firestore and using something like Mongo

Reply:
fo sure 👍🏻